### ISSUE TITLE
java.io.FileNotFoundException android-distance.json

### ISSUE THREAD

**Issue by TinTran64** [AltBeacon/android-beacon-library#1183]
I'm seeing this warning in my application log is it safe to ignore?

```
java.io.FileNotFoundException: https://s3.amazonaws.com/android-beacon-library/android-distance.json
at com.android.okhttp.internal.huc.HttpURLConnectionImpl.getInputStream(HttpURLConnectionImpl.java:255)
at com.android.okhttp.internal.huc.DelegatingHttpsURLConnection.getInputStream(DelegatingHttpsURLConnection.java:211)
at com.android.okhttp.internal.huc.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:30)
at org.altbeacon.beacon.distance.DistanceConfigFetcher.request(DistanceConfigFetcher.java:94)
at org.altbeacon.beacon.distance.ModelSpecificDistanceUpdater.doInBackground(ModelSpecificDistanceUpdater.java:28)
at org.altbeacon.beacon.distance.ModelSpecificDistanceUpdater.doInBackground(ModelSpecificDistanceUpdater.java:16)
at android.os.AsyncTask$3.call(AsyncTask.java:394)
at java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
at java.lang.Thread.run(Thread.java:1012)

Cannot updated distance models from online database at java.io.FileNotFoundException: https://s3.amazonaws.com/android-beacon-library/android-distance.json```

**Comment by davidgyoung** [comment]
Yes, please ignore this issue for now -- we are changing the way these remote updates happen which is why the file not found is happening.  There is zero consequence for newly developed apps because the data bundled with the library is  currently identical to what is available on the server.

**Comment by davidgyoung** [comment]
Fixed in #1190

**Comment by AltenHu** [comment]
> Fixed in #1190

@davidgyoung - Has this fix been released in the latest version [2.20.6](https://mvnrepository.com/artifact/org.altbeacon/android-beacon-library/2.20.6)?
